src/main/scala/io/iohk/ethereum/jsonrpc/JsonRpcController.scala

```diff
@@ -261,6 +261,18 @@ class JsonRpcController(
       handle[SignRequest, SignResponse](personalService.sign, req)(eth_sign, personal_sign)
     case req @ JsonRpcRequest(_, "eth_getStorageRoot", _, _) =>
       handle[GetStorageRootRequest, GetStorageRootResponse](ethService.getStorageRoot, req)
+    case req @ JsonRpcRequest(_, "eth_getRawTransactionByHash", _, _) =>
+      handle[GetRawTransactionByHashRequest, GetRawTransactionByHashResponse](ethService.getRawTransactionByHash, req)
+    case req @ JsonRpcRequest(_, "eth_getRawTransactionByBlockHashAndIndex", _, _) =>
+      handle[GetRawTransactionByBlockHashAndIndexRequest, GetRawTransactionByBlockHashAndIndexResponse](
+        ethService.getRawTransactionByBlockHashAndIndexRequest,
```

Comment:
minor: unnecessary `Request` suffix in method name (same below)

Reply:
Renamed (no Request suffix). I was misled by getTransactionXXX functions that have Request suffix as well.

Small impro to Scaladoc: https://github.com/input-output-hk/mantis/pull/720/commits/f8ff637b7c6862988482b318346b2b9b462b7513